api/.import-restrictions

```diff
@@ -0,0 +1,11 @@
+rules:
+  # Enforce that our APIs do not depend on internal packages.
```

Comment:
This blocks everything globally and then we only enable locally when we need more.

This is safer than the other way around (allowing to much globally and then blocking in v1beta2 packages)